jekyll/_cci2/server/operator/using-metrics.adoc

```diff
@@ -0,0 +1,56 @@
+---
+version:
+- Server v4.x
+- Server Admin
+---
+= Using metrics
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Learn about CircleCI server installation metrics collection and monitoring.
+:icons: font
+:toc: macro
+:toc-title:
+
+Metrics such as CPU or memory usage and internal metrics can help you manage and debug your server installation in many ways, including:
+
+* Quickly detecting incidents and abnormal behavior.
+* Dynamically scaling compute resources.
+* Retroactively understanding infrastructure-wide issues.
+
+toc::[]
+
+[#metrics-collection]
+== Metrics collection
+
+[#scope]
+=== Scope
+Your CircleCI server installation collects a number of metrics and logs by default, which can be useful in monitoring the health of your system and debugging issues with your installation.
+
+[#telegraph]
+=== Telegraf
+Most services running on server report StatsD metrics to the https://www.influxdata.com/time-series-platform/telegraf/[Telegraf] pod running in server.
+The configuration is fully customizable, so you can forward your metrics from Telegraf to any output supported by Telegraf through https://docs.influxdata.com/telegraf/v1.17/plugins/#output-plugins[output plugins]. By default, it will provide a metrics endpoint for Prometheus to scrape.
+
+[#use-telegraf-to-forward-metrics-to-datadog]
+=== Use Telegraf to forward metrics to Datadog
+The following example shows how to configure Telegraf to output metrics to Datadog:
+
+. Open the management console dashboard and select **Config** from the menu bar.
+. Locate the **Custom Telegraf config** section under **Observability and monitoring**. There is an editable text window where you can configure plugins for forwarding Telegraf metrics for your server installation.
+. To forward to Datadog, add the following code, substituting `my-secret-key` with your Datadog API key:
++
+[source,shell]
+----
+[[outputs.datadog]]
+  ## Replace "my-secret-key" with Datadog API key
+  apikey = "my-secret-key"
+----
+
+For more options, see the https://docs.influxdata.com/telegraf/v1.17/plugins/#output-plugins[Influxdata docs].
```

Comment:
Maybe better to say "Telegraf docs" as opposed to Influxdata docs? Mostly because "Influxdata" wasn't mentioned anywhere prior